src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const Price = 40;
+  const PriceForWeek = 50;
+  const PriceForThreeDays = 20;
```

Comment:
The constant names should be more descriptive and should follow the UPPER_CASE naming convention for constants. Consider renaming `Price` to `DAILY_RENTAL_RATE`, `PriceForWeek` to `LONG_TERM_DISCOUNT`, and `PriceForThreeDays` to `SHORT_TERM_DISCOUNT` to clearly reflect their purpose in the context of rental cost calculation.